### PR TITLE
Add French Elections Memos reuse in homepage

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: src/translations/fr/homepage.json [hunk 1]
```diff
@@ -23,11 +23,14 @@
   "how.bloc4.title": "4. Publication",
   "how.title": "Fonctionnement",
   "mission_statement": "Quand les services que nous utilisons au quotidien changent leurs règles du jeu, nous sommes les premiers concernés.",
+  "reuses.memos-elections-fr.author": "Reset",
+  "reuses.memos-elections-fr.subtitle": "Suivi du comportement des plateformes sur des périodes électorales.",
```
Comment (MattiSG) [inline src/translations/fr/homepage.json:27]: All phrasings are in the active phrasing and the present tense.
```suggestion
  "reuses.memos-elections-fr.subtitle": "Suivi du comportement des plateformes sur des périodes électorales.",
```
```suggestion
  "reuses.memos-elections-fr.subtitle": "Suivi du comportement des plateformes sur des périodes électorales.",
```
```suggestion
  "reuses.memos-elections-fr.subtitle": "Suivi du comportement des plateformes sur des périodes électorales.",
```
```suggestion
  "reuses.memos-elections-fr.subtitle": "Suit le comportement des plateformes en période électorale.",
```

File: src/translations/en/homepage.json [hunk 2]
```diff
@@ -23,11 +23,14 @@
   "how.bloc4.title": "4. Publishing",
   "how.title": "How it works",
   "mission_statement": "When the services we use every day change their rules, we should be the first ones to know.",
+  "reuses.memos-elections-fr.author": "Reset",
+  "reuses.memos-elections-fr.subtitle": "Tracking platforms behaviour over electoral periods.",
```
Comment (MattiSG) [inline src/translations/en/homepage.json:27]: All phrasings are in the active tense.
```suggestion
  "reuses.memos-elections-fr.subtitle": "Tracks major platforms behaviour over the French 2022 elections.",
```
```suggestion
  "reuses.memos-elections-fr.subtitle": "Tracking platforms behaviour over electoral periods.",
```
```suggestion
  "reuses.memos-elections-fr.subtitle": "Tracking platforms behaviour over electoral periods.",
```
```suggestion
  "reuses.memos-elections-fr.subtitle": "Tracking platforms behaviour over electoral periods.",
```

Comment (MattiSG) [inline src/translations/en/homepage.json:27]: All phrasings are in the active tense.
```suggestion
  "reuses.memos-elections-fr.subtitle": "Tracks major platforms behaviour over the French 2022 elections.",
```

**Review by MattiSG — CHANGES_REQUESTED** [review]
Why are the illustrations not full width? 🙂 This looks surprisingly small compared to the other ones.

Screen Shot 2022-06-12 at 22 50 31

**Comment by clementbiron** [comment]
@MattiSG the current format of the illustration allow to show the content of a memo.

With fullwidth it gives a less good idea of the content and there is a an unsightly cut between the header and the intro of the memo. Thiis version makes me want to show only the black header. What do you think ?

image

**Review by MattiSG — CHANGES_REQUESTED** [review]
Following our synchronous rebase and force-push, you should cherry-pick 416bfa3f6babe07a7087f5bd028f658d161831d5 to this branch.

**Comment by MattiSG** [comment]
> This version makes me want to show only the black header. What do you think ?

I agree with the weird cut not looking good, but I would be more in favour of showing only the contents of the newsletter instead, as the black header looks too much like the main page layout.

**Comment by clementbiron** [comment]
Good idea! 

I pushed my proposal. 
image

**Comment by MattiSG** [comment]
Great! I see the screenshot still has the current wording. Please make sure you include the changes for wording that had already been suggested 🙂

**Review by MattiSG — APPROVED** [review]
🚀

**Pull request merged**